compiler/test/runnable/test15985.d

```diff
@@ -0,0 +1,15 @@
+// https://issues.dlang.org/show_bug.cgi?id=15985
+
+void ff()()
```

Comment:
Because there are two possible emission behaviours in `needsCodegen`, we definitely need to test the this with `-allinst` as well.

Reply:
`PERMUTE_ARGS: -allinst` if I remember correctly will test both with and without flag.

Reply:
added -allinst